dev/Breadcrumb/BreadcrumbBar.xaml

```diff
@@ -336,13 +338,28 @@
                             </Grid>
                             
                         </Button>
+
+                        <ContentPresenter x:Name="PART_LastItemContentPresenter"
+                                          Visibility="Collapsed"
+                                          Content="{TemplateBinding Content}"
```

Comment:
>Content [](start = 68, length = 7)

It doesn't make sense to me that PART_LastItemContentPresenter and PART_EllipsisDropDownItemContentPresenter both template bind their content to the controls Content. I suspect this would crash if they were both present.

Reply:
It looks like PART_ItemContentPresenter also template binds the same property..

---
In reply to: [605843861](https://github.com/microsoft/microsoft-ui-xaml/pull/4666#discussion_r605843861) [](ancestors = 605843861)

Reply:
it's expected that both should never be present at the same time, a BreadcrumbItem can only show the PART_LastItemContentPresenter, PART_EllipsisDropDownItemContentPresenter  or PART_ItemButton one at a time

Reply:
Hmm, okay.